milestone2.ts

```diff
@@ -74,43 +74,79 @@ const userData: Volunteer[] = [
     city: "SF",
     state: "CA",
   },
+
+  {
+    name: "Luna",
+    age: 66,
+    email: "luna@email.com",
+    position: "volunteer",
+    city: "SLO",
+    state: "CA",
+  }
 ];
 
 // Questions Start Here
 
 // Question 1: Lambda Functions
-/* Define a new lambda function that finds the average age of the people in the data. 
+/* Define a new lambda function that finds the average age of the users in the data. 
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data) =>{
+  let tot = 0;
+  for (let i = 0; i < data.length; i++){
+    tot += data[i].age
+  }
+  return tot / data.length
 
-//console.log(findAverage(userData));
+} // Define lambda function here
+
+console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+type GetCity = (data: Volunteer[]) => number;
+let findIndexAns: GetCity = (data) =>{ 
+  //data.findIndex(city => city === "SF");
```

Comment:
You were so close to getting this working!
```suggestion
  //data.findIndex(vol=> vol.city === "SF");
```